model.py

```diff
@@ -0,0 +1,101 @@
+# generated by datamodel-codegen:
+#   filename:  application_schema.yaml
+#   timestamp: 2024-03-12T19:55:46+00:00
+
+from __future__ import annotations
+
+from datetime import date
+from typing import List
+
+from pydantic import AnyUrl, AwareDatetime, BaseModel, ConfigDict, Field, RootModel
+
+
+class Link(BaseModel):
+    model_config = ConfigDict(
+        extra='forbid',
+    )
+    title: str
+    url: AnyUrl
+
+
+class ReductionOfTheAttackSurfaceComponent(BaseModel):
```

Comment:
I would not put the hole model in the collector. Only the one needed (in this case threat modeling)

Reply:
simplified the model.